### ISSUE TITLE
[jit][script] Support tuple unpacking assignments to class variables

### ISSUE THREAD

**Issue by alanhdu** [pytorch/pytorch#28303]
## 🐛 Bug

Right now, `torch.jit.script` does not seem to support tuple unpacking assignments to class variables (e.g. `self.a, self.b = (1, 2)`). Instead, you have to run something like:

```python
tup = (1, 2)
self.a = tup[0]
self.b = tup[1]
```

This isn't a big deal because of the easy workaround, but it is kind of a papercut.

## To Reproduce

Using Pytorch 1.3, running the following script:

```python
import torch
import torch.nn as nn

class A(nn.Module):
    def __init__(self):
        super().__init__()
        self.a = 1
        self.b = 2
    def forward(self, inputs):
        self.a, self.b = (1, 2)
        return inputs

a = torch.jit.script(A())
```

fails with

```
Traceback (most recent call last):
  File "test.py", line 14, in <module>
    a = torch.jit.script(A())
  File "/opt/anaconda/envs/test/lib/python3.6/site-packages/torch/jit/__init__.py", line 1203, in script
    return torch.jit.torch.jit._recursive.recursive_script(obj)
  File "/opt/anaconda/envs/test/lib/python3.6/site-packages/torch/jit/_recursive.py", line 173, in recursive_script
    return copy_to_script_module(mod, overload_stubs + stubs)
  File "/opt/anaconda/envs/test/lib/python3.6/site-packages/torch/jit/_recursive.py", line 95, in copy_to_script_module
    torch.jit._create_methods_from_stubs(script_module, stubs)
  File "/opt/anaconda/envs/test/lib/python3.6/site-packages/torch/jit/__init__.py", line 1423, in _create_methods_from_stubs
    self._c._create_methods(self, defs, rcbs, defaults)
RuntimeError: 
lhs of assignment must be a variable, subscript, or starred expression:
at test.py:10:8
    def forward(self, inputs):
        self.a, self.b = (1, 2)
        ~~~~~~ <--- HERE
        return inputs
```

## Expected behavior

For the scriptification to pass without errors.

## Environment

```
PyTorch version: 1.3.0
Is debug build: No
CUDA used to build PyTorch: None

OS: Ubuntu 18.04.3 LTS
GCC version: (Ubuntu 8.3.0-6ubuntu1~18.04.1) 8.3.0
CMake version: version 3.14.5

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip] numpy==1.16.4
[pip] numpy-quaternion==2019.7.23.15.26.49
[pip] torch==1.3.0
[conda] cpuonly                   1.0                           0    pytorch
[conda] mkl                       2019.4                      243  
[conda] pytorch                   1.3.0               py3.6_cpu_0  [cpuonly]  pytorch
```

cc @suo

**Comment by eellison** [comment]
To fix this, I would recommend gdb'ing through the compilation process of 
```
tup = (1, 2)
self.a = tup[0]
self.b = tup[1]
``` 
and then gdb'ing through 
```
self.a, self.b = (1, 2)
```
and try to make the compilation paths line up.

**Comment by ezyang** [comment]
I was affected by this when porting the seq2seq in https://github.com/stanfordnlp/stanza to TorchScript

**Comment by calvinytong** [comment]
Hey, I believe this issue is fixed in 1.9, so we can probably close this issue
